example/test_problem/Hydro/GrackleTest/generate_make.sh

```
@@ -0,0 +1,5 @@
+# This script should run in the same directory as configure.py
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)